components/CommunityActionSection/CommunityActionSection.js

```diff
@@ -18,10 +19,18 @@ export default function CommunityActionSection(props) {
         With many choices to pick from, we’ll help you find a group or class
         that’s right for you!
       </Box>
-      <Button onClick={props.handleOnClick} rounded={true} mb="base">
-        {/* Allows for Group Hub titles to be passed in */}
-        {`Search All ${props?.title ? props?.title + ' ' : ''}Groups & Classes`}
-      </Button>
+      
+      {classesHub &&(
+        <Button onClick={props.handleOnClick} rounded={true} mb="base">
+          Search All Groups & Classes
```

Comment:
Good idea with checking for a "Classes" hub. In order to help keep things a bit more "maintainable", we've actually gotten approval to make the default to just be "Search All Groups & Classes". So we can actually just set the test to be hardcoded to be: `Search All Groups & Classes`

Reply:
Oh, easy, will do, thanks!